.github/workflows/os-smoke-test.yml

```diff
@@ -6,16 +6,19 @@ on:
   pull_request:
     branches: [ main ]
 
+env:
+  JAVA_TOOL_OPTIONS: -XX:+TieredCompilation -XX:TieredStopAtLevel=1
+
 jobs:
   smoke-test:
     # This name is hard-referenced from bors.toml
```

Comment:
@npepinpe Reminder to update bors.toml as well

Reply:
I was totally going to that before you said it, of course... :smile:

Reply:
Also, I didn't even request the review yet and in 2 minutes I get a review? :thinking:

Reply:
Haha, i was keeping an eye on this PR because I came across the comment to update bors.toml a few days ago and though "huh, that's easy to forget" :smile: